synapse/storage/schema/__init__.py

```diff
@@ -45,10 +45,11 @@
 Changes in SCHEMA_VERSION = 65:
     - MSC2716: Remove unique event_id constraint from insertion_event_edges
       because an insertion event can have multiple edges.
+    - Remove unused tables `user_stats_historical` and `room_stats_historical`.
 """
 
 
-SCHEMA_COMPAT_VERSION = 60  # 60: "outlier" not in internal_metadata.
+SCHEMA_COMPAT_VERSION = 61  # 60: "outlier" not in internal_metadata.
```

Comment:
Does the comment need updating?

Reply:
I kind of wish we would do the same as above, where we kept historical descriptions and had a little more room to write something. Though maybe I'm missing the point of this comment.

I assume it's to describe why the value was last bumped.

Reply:
Should I bump the comment to something like # 61: Remove unused tables `user_stats_historical` and `room_stats_historical` ? I didn't want to bump or delete it because I wasn't entirely sure why it was there ;) but it seem that the consensus is that it can go...

Reply:
We talked about not keeping historical descriptions before, but only a summary.  See https://github.com/matrix-org/synapse/pull/10903#discussion_r716570726 :)